test/integration/utils/utils.go

```diff
@@ -32,3 +35,19 @@ func GetTestProjectFromSetup(t *testing.T, idx int) string {
 	}
 	return projectIDs[idx]
 }
+
+var (
+	// TGKEVerify Exempt Resources. e.g. google_container_cluster.primary
+	verifyExemptResources = []string{}
+)
+
+func TGKEVerify(t *testing.T, b *tft.TFBlueprintTest, assert *assert.Assertions) {
+	_, ps := b.PlanAndShow()
+	for _, r := range ps.ResourceChangesMap {
+		if slices.Contains(verifyExemptResources, r.Address) {
```

Comment:
Should we consider matching based on suffix (so we can provide a snippet like `google_container_cluster.primary`) otherwise we will to write the full address like `module.example.module.gke.google_container_cluster.primary`

Reply:
Sure, I think the combo of `slices.ContainsFunc() and `strings.HasSuffix()` should do it.